.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing Guidelines
+
+There are a number of ways you can help by Contributing to this project, Here are some:
+
+- Improve the docs and point out suggestions / improvements
+- Send pull requests if you can fix a bug.
+- Always be polite in your approach.
```

Comment:
@puranjayjain this is a nice addition :) What if we said:

There are a number of ways you can help by contributing to this project:

- Improve the docs and point out suggestions / improvements
- Send pull requests if you can fix a bug
- Be polite and ask questions if you're uncertain of how to help
- Escalate issues and pull requests with a :+1: on the first comment instead of adding a new comment.
- Mention the project contributors and/or the repo owner for a faster response to your issue or pull request